fixtures/bright/xbabytw.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "xbabytw",
+  "shortName": "baby",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["jens"],
+    "createDate": "2025-03-10",
+    "lastModifyDate": "2025-03-10"
+  },
+  "links": {
+    "video": [
+      "https://stageline.no/lyd/stativ/streaming-og-podcast-tilbehør/bright-xbaby-color-rgbw-led-fresnel-black-powercon-xline/888723"
+    ]
+  },
+  "physical": {
+    "DMXconnector": "5-pin"
+  },
+  "wheels": {
+    "Color Wheel": {
+      "slots": [
+        {
+          "type": "Open"
+        }
+      ]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "fineChannelAliases": ["Dimmer fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Color Temperature": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "warm",
+        "colorTemperatureEnd": "cold"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 255,
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Open"
+      }
+    },
+    "Color Wheel": {
+      "capability": {
+        "type": "WheelSlot",
+        "slotNumber": 1
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "5 channel",
```

Comment:
The manual lists 2/3/6/9ch modes.  16-bit dimming is only available as channels 5-6 of the 9ch mode.  Color temp is channel 8, and strobe (not working here) is channel 7 -- but the virtual color wheel is only available in 2ch mode.

So this mapping isn't usable for any mode, as it stands.  The documentation is good, so someone could fix this, but you'd basically have to start from scratch.